examples/dojo/js/todo/empty.js

```diff
@@ -0,0 +1 @@
+define(function () {});
```

Comment:
For: https://github.com/asudoh/todomvc/blob/dojo110/examples/dojo/js/main.js#L29-L34 Will add a comment for more explanation.